pkg/cmdimpl/status.go

```diff
@@ -0,0 +1,110 @@
+package cmdimpl
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/weaveworks/weave-gitops/pkg/fluxops"
+	"gopkg.in/yaml.v2"
+
+	"github.com/weaveworks/weave-gitops/pkg/utils"
+)
+
+// Status provides the implementation for the wego status application command
+func Status(args []string, allParams AddParamSet) error {
+
+	allParams.Name = args[0]
+
+	// verify the app is in the wego apps folder
+	appPath, err := utils.GetWegoAppPath(allParams.Name)
+	if err != nil {
+		return fmt.Errorf("error getting path for app [%s] \n", args[0])
+
+	}
+
+	if !utils.Exists(appPath) && allParams.Name != "wego" {
+		return fmt.Errorf("app provided does not exists on apps folder [%s]\n", args[0])
+	}
+
+	deploymentType, err := getDeploymentType(allParams.Namespace, allParams.Name)
+	if err != nil {
+		return fmt.Errorf("error getting deployment type [%s]", err)
+	}
+
+	err = printOutLatestSuccessfulDeploymentType(allParams.Namespace, allParams.Name, deploymentType)
+	if err != nil {
+		return fmt.Errorf("error on latest deployment time [%s]", err)
+	}
+
+	cmd := fmt.Sprintf(`get all -A %s`,
+		allParams.Name)
+	_, err = fluxops.CallFlux(cmd)
+	return err
+}
+
+type Yaml struct {
+	Status struct {
+		Conditions []struct {
+			LastTransitionTime string `yaml:"lastTransitionTime"`
+		} `yaml:"conditions"`
+	} `yaml:"status"`
+}
+
+func printOutLatestSuccessfulDeploymentType(namespace, appName string, deploymentType DeploymentType) error {
+
+	c := fmt.Sprintf(`kubectl \
```

Comment:
I don't understand the motivation for all the line continuations here.

Reply:
It is easy to visualize the whole command at once. This might not be as large as others but I'm just trying to get use to this pattern 🙂

Reply:
Okay :-)